@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -2,11 +2,13 @@
 	<section
 		ref="rootEl"
 		class="kv-carousel tw-overflow-hidden tw-w-full"
+		:class="{ 'lg:tw-relative': asideControls }"
 		aria-label="carousel"
 	>
 		<!-- Carousel Content -->
 		<div
 			class="tw-flex tw-gap-x-4"
+			:class="{ 'tw-mx-auto lg:tw-w-[82%]': asideControls }"
```

Comment:
We've been trying not to use the JIT TW classes. An alternative here is to use a custom class and CSS.